app/controllers/services/service_bindings_controller.rb

```diff
@@ -25,6 +25,20 @@ def read(guid)
       object_renderer.render_json(self.class, obj, @opts)
     end
 
+    get '/v2/service_bindings/:guid/parameters', :parameters
+
+    def parameters(guid)
+      binding = find_guid(guid)
+      raise CloudController::Errors::ApiError.new_from_details('ServiceBindingNotFound', guid) unless binding.v2_app.present?
```

Comment:
Is line 32 dead code? I couldn't get a controller unit test to fail when I deleted it in my local environment.

Opening up a chore to take a look (https://www.pivotaltracker.com/story/show/154875456). If it is unneeded, we'll open a PR to delete it down the line so we don't need to rebase all the PRs that build off of this one.

Reply:
`/v2/service_bindings` only support the web process type and filters out all service bindings to any other type. After talking with Zach from CAPI he said that the general idea is that bindings should be per App, not per process and this guard is there on purpose. So we have kept this in and added a test for the line.